tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -1577,6 +1577,10 @@ def test_OnDiskDataset_load_graph():
         with open(yaml_file, "w") as f:
             f.write(yaml_content)
 
+        # Check if the CSCSamplingGraph.edge_attributes loaded.
+        dataset = gb.OnDiskDataset(test_dir).load()
+        assert dataset.graph.edge_attributes is not None
```

Comment:
how about checking the content of `edge_attribute`?

Reply:
We have already verified the `edge_attributes` in the unit test `test_cscsampling_graph`. Do you believe it's necessary to check it again here?